src/main/scala/machines/Internal.scala

```diff
@@ -4,3 +4,33 @@ import regex._
 import dfa._
 
 // TODO: Add your code below
+
+given Conversion[Char, RegularLanguage] = n => Character(n)
+
+// given Conversion[String, RegularLanguage] = n => if (n.length == 1) Character(n(0)) else (Concat(Character(n(0)), (n.slice(1, n.length))))
+given Conversion[String, RegularLanguage] = n => n.toList.foldRight(Concat(Epsilon, Epsilon))(Concat(_,_))
+
+
```

Comment:
You don't seem to have a DFA implicit convertor. Which, I don't either. High five!